### PR TITLE
Add starter recipes / themes docs

### PR DESCRIPTION
**Pull request by deanmarcussen**
Some documentation about what comes oob with Orchard Core Recipes and Themes.

Not exhaustive, but some information to get people going a bit easier

### CODE REVIEW

File: src/docs/getting-started/starter-recipes.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+## OrchardCore.Application.Cms.Core.Targets
+
+### Empty Recipe
+
+The Empty recipe enables content management features, but does not set a current theme.
+You can use this recipe when starting Orchard Core in Decoupled Mode,
+or when building your own theme.
+
+Alternativaley you can start with another recipe,
```
Comment (agriffard) [inline src/docs/getting-started/starter-recipes.md]: Alternativaley => Alternatively

File: src/docs/getting-started/starter-recipes.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+- A blog, and a first blog post, based on the `ListPart`
+- Liquid templates, in the TheBlogTheme source code
+- Bootstrap
+
+### TheAgencyTheme and Agency Recipe
+
+The Agency recipe sets up a range of content types, and widgets, the initial content,
+and sets the current theme to TheAgencyTheme.
+
+TheAgencyTheme is based of the [Start Bootstrap Agency Theme](https://startbootstrap.com/themes/agency/)
```
Comment (agriffard) [inline src/docs/getting-started/starter-recipes.md]: based of => based on

File: src/docs/getting-started/starter-recipes.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- A LandingPage, based on the `BagPart`
+- Liquid templates, in TheAgencyTheme source code, and Templates feature
+- Bootstrap
+
+### ComingSoon Recipe and TheComingSoonTheme
+
+This recipe sets up a range of Content Types, and Widgets, and the initial content of TheComingSoonTheme.
+It also includes Email, Recaptcha, Forms, Workflows and User Registration Forms.
+
+TheComingSoon theme is based of the [Start Bootstrap Coming Soon Theme](https://startbootstrap.com/themes/coming-soon/)
```
Comment (agriffard) [inline src/docs/getting-started/starter-recipes.md]: based of

File: src/docs/getting-started/starter-recipes.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    it suits your requirements.
+
+## OrchardCore.Application.Cms.Targets
+
+### TheBlogTheme and Blog Recipe
+
+The Blog recipe sets up a range of content types, and widgets, the initial content,
+and sets the current theme to the TheBlogTheme.
+
+TheBlogTheme is based of the [Start Bootstrap Clean Blog Theme](https://startbootstrap.com/themes/clean-blog/)
```
Comment (agriffard) [inline src/docs/getting-started/starter-recipes.md]: based of

File: src/docs/getting-started/starter-recipes.md (hunk 2 above)
Comment (deanmarcussen) [reply src/docs/getting-started/starter-recipes.md]: Done / all

**Pull request merged**